pkg/kubelet/checkpoint/checkpoint.go

```diff
@@ -94,6 +93,7 @@ func LoadPods(cpm checkpointmanager.CheckpointManager) ([]*v1.Pod, error) {
 	checkpointKeys, err = cpm.ListCheckpoints()
 	if err != nil {
 		glog.Errorf("Failed to list checkpoints: %v", err)
+		return nil, err
```

Comment:
i think not returning an error here was intentional.

/assign @vikaschoudhary16 
as per https://github.com/kubernetes/kubernetes/pull/56040

Reply:
@neolit123 thanks for the ref. It does look like intentional, I am curious about the reason.